api/views/workflow.py

```diff
@@ -40,10 +45,92 @@ class SourceViewSet(viewsets.ReadOnlyModelViewSet):
     ordering = ('id', )
     ordering_fields = ('long_title', )
     serializer_class = SourceSerializer
+    permission_classes = [DjangoModelPermissionsOrAnonReadOnly, ]
+
+    queryset = Source.objects.none()  # Required for DjangoModelPermissions
```

Comment:
One would think. I didn't actually try it without though; just trusted the [docs](http://www.django-rest-framework.org/api-guide/permissions/#djangomodelpermissions):
> If you're using this permission with a view that uses an overridden get_queryset() method there may not be a queryset attribute on the view. In this case we suggest also marking the view with a sentinel queryset, so that this class can determine the required permissions. For example: